src/pynxtools/nomad/schema.py

```diff
@@ -1258,6 +1276,7 @@ def normalize_atom_probe(self, archive, logger):
     ].section_cls
     super(current_cls, self).normalize(archive, logger)
     # temporarily disable extra normalisation step
+    return
```

Comment:
This is currently bypassed as there are still flaws in the original implementation in that it does not match with the new way where data are stored using the new definitions, I do not want this to be a blocker for the release for Christoph.